### PR TITLE
v5.0.0 bake constants and static content

### PR DESCRIPTION
**Pull request by coury-clark**
This pull request was automatically generated by the release-tool.
Bake constants and static content into version v5.0.0.
## Test Plan:
N/A

### CODE REVIEW

File: client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import styles from './BatchChangesListIntro.module.scss'
  * Ie. After 5.0 is cut, this should be bumped to 5.1, and so on. This ensures we
  * always render the right changelog.
  */
-const CURRENT_VERSION = '4.4'
+const CURRENT_VERSION = '5.0'
```
Comment (coury-clark) [inline client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:16]: @eseliger This is the PR for the release branch, could you please verify this is correct for in app changelog? (including the boolean below)

Comment (BolajiOlajide) [reply client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:16]: Ha! Okay, now I'm more confused, lol. In the previous PR I saw this was 5.1

Comment (BolajiOlajide) [reply client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:16]: The boolean and the `CURRENT_VERSION` variable are correct. We don't have anything to display so it should be `false`.

Comment (eseliger) [reply client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:16]: yep!

**Review by camdencheek — APPROVED** [review]

**Review by DaedalusG — APPROVED** [review]

**Review by eseliger — APPROVED** [review]
That's great, so happy this is all automated now!

**Pull request merged**